kinto2xml/exporter.py

```diff
@@ -53,22 +74,33 @@ def write_addons_items(xml_tree, records):
       </prefs>
     </emItem>
     """
+    if not records:
+        return
+
     emItems = etree.SubElement(xml_tree, 'emItems')
     for item in records:
-        if item.get('enabled', True):
+        if item.get('enabled', True) and is_related_to(item, app_id):
```

Comment:
instead of test `enabled` here, I would iterate on records before providing them to this function (since it concerns every kind of entry)